cmd/multiFaToVcf/multiFaToVcf.go

```diff
@@ -1,17 +1,18 @@
 // Command Group: "Data Conversion"
 
-// Generates a VCF file from an input pairwise multiFa alignment with the first entry as the reference
+// Generates a VCF file from an input pairwise or three-way multiFa alignment with the first entry as the reference
 package main
 
 import (
 	"flag"
 	"fmt"
+	"log"
+
 	"github.com/vertgenlab/gonomics/convert"
 	"github.com/vertgenlab/gonomics/exception"
 	"github.com/vertgenlab/gonomics/fasta"
 	"github.com/vertgenlab/gonomics/fileio"
 	"github.com/vertgenlab/gonomics/vcf"
-	"log"
 )
 
 func multiFaToVcf(inFile string, chr string, outFile string, substitutionsOnly bool, retainN bool) {
```

Comment:
In the main function of this program, we'll need to check whether the input FASTA has 2 or 3 entries and then send it to the corresponding function in convert.go

Reply:
Line 22/23 of this file multiFaToVcf.go is where we'll need to check the length of the fasta (variable is called f) and send it to the right convert function (Pairwise or ThreeWay)